stable/kong/templates/admission-webhook.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if .Values.ingressController.admissionWebhook.enabled }}
+{{- $cn := printf "%s-validation-webhook.%s.svc" ( include "kong.fullname" . ) .Release.Namespace }}
+{{- $ca := genCA "kong-admission-ca" 3650 -}}
+{{- $cert := genSignedCert $cn nil nil 3650 $ca -}}
```

Comment:
+1 on self-signed cert be need configuration options to make this configurable by the end user well.
What are your thoughts @rifelpet?

We could add it later if someone actually asks for it. I'm not sure if users want it (user-provided key-pair) or not, hence the question.

Reply:
Agreed, my team won't have user-provided key-pairs but it should be trivial to add support for it in a backwards-compatible way so I decided not to implement it myself. If someone else wants to add it, they're free to do so.